examples/containers/run.sh

```diff
@@ -20,14 +20,9 @@ COUCHBASE_SERVER_IMAGE=${COUCHBASE_SERVER_IMAGE:-couchbase/server:7.0.2}
 
 DOCKER_USER=${DOCKER_USER:-couchbase}
 DOCKER_TAG=${DOCKER_TAG:-v1}
-CMOS_IMAGE=${CMOS_IMAGE:-$DOCKER_USER/observability-stack:$DOCKER_TAG}
-
 # Ensure we build the container locally first otherwise make
```

Comment:
Left in?

Reply:
It's required to set the variable to compose if you do want to override it. Just making it explicit what is important and also the defaults.

Reply:
Sorry I mean the comment "# Ensure we build the container locally first..."

Reply:
Yeah I wanted to document it here as well, someone might just look at the run.sh script.

Reply:
Ah i see, maybe rewrite as "Ensure you have built the container... "